### PR TITLE
feat: add net8 target

### PR DESCRIPTION
**Pull request by mirceaarhip**

### CODE REVIEW

File: src/Directory.Build.props [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
+	<PropertyGroup>
+        <CoreFrameworks>|net6.0|net8.0|</CoreFrameworks>
+		<WinFrameworks>|net6.0-windows|net8.0-windows|</WinFrameworks>
+    </PropertyGroup>
     <PropertyGroup>
         <Version>6.0.0</Version>
         <RoslynPackageVersion>4.5.0-2.22527.10</RoslynPackageVersion>
         <WarningsAsErrors>true</WarningsAsErrors>
-        <UseWPF Condition="$(TargetFramework) == 'net6.0-windows'">true</UseWPF>
+        <UseWPF  Condition="$(WinFrameworks.Contains('|$(TargetFramework)|'))">true</UseWPF>
```
Comment (aoltean16) [inline src/Directory.Build.props:10]: Why not `$(TargetFramework).Contains('windows')`. Makes it future proof, and we dont need the extra properties to maintain

File: src/Test/TestCases.Activities/MethodInvokeTests.cs [hunk 2]
```diff
@@ -189,7 +189,7 @@ public void CallOverloadedWriteLine()
             catch (Exception exception)
             {
                 Dictionary<string, string> exceptionProperty = new Dictionary<string, string>();
-                exceptionProperty.Add("Message", string.Format("Ambiguous match found."));
+                exceptionProperty.Add("Message", string.Format("Ambiguous match found"));
```
Comment (aoltean16) [inline src/Test/TestCases.Activities/MethodInvokeTests.cs:192]: Nit: for future, better to keep PR's focused on one thing.

File: src/Directory.Build.props (hunk 1 above)
Comment (mirceaarhip) [reply src/Directory.Build.props:10]: it's a matter of preferences for me is clear this way but if you insist i can make the change

File: src/Test/TestCases.Activities/MethodInvokeTests.cs (hunk 2 above)
Comment (mirceaarhip) [reply src/Test/TestCases.Activities/MethodInvokeTests.cs:192]: it is one thing :D In net8 the test fails because the message no longer contains .

**Comment by mirceaarhip** [comment]
daca nu scoatem TFM net6, nu am facut nimic ca targetam si net8
[12:16](https://uipath-product.slack.com/archives/C06HDUYQ5RD/p1707387417045499)
nu obtinem nimic pe plus, doar buguri si batai de cap
New
[12:17](https://uipath-product.slack.com/archives/C06HDUYQ5RD/p1707387444216379)
putem discuta despre a scoate net6 dar nu cred ca vrem sa avem discutia asta
[12:17](https://uipath-product.slack.com/archives/C06HDUYQ5RD/p1707387448831279)
deci cel mai simplu, let it go

**Pull request closed without merging**